salt/output/json_out.py

```diff
@@ -54,7 +54,7 @@ def output(data):
     '''
     try:
         if 'output_indent' not in __opts__:
-            return json.dumps(data, default=repr, indent=4)
+            return json.dumps(data, default=repr, indent=4, encoding='latin1')
```

Comment:
Hello, 

if I encode with "latin1" I get results as follow: "SDK de comprobaci**\u00a2**n de Visual Studio 2012 - esn". If I use for the same minion utf8 I get the error as described. 
In addidtion it looks like windows encodes anything in utf16 (https://en.wikipedia.org/wiki/Unicode_in_Microsoft_Windows).

Best Malte